### ISSUE TITLE
Add limitation to WORM CVO doc

### ISSUE THREAD

**Issue by netapp-joey** [NetAppDocs/occm#543]
Page: [WORM storage](https://docs.netapp.com/us-en/occm/concept_worm.html)

Hello. There is an important limitation with WORM on CVO that I'd like to request we add to the limitation section of this page. Requested addition:

"- WORM storage in Cloud Volumes ONTAP operates under a “trusted storage administrator” model. While WORM files are protected from alteration or modification, volumes can be deleted by the cluster administrator even if those volumes contain unexpired WORM data."

Thanks.

**Comment by netapp-bcammett** [comment]
Hi Joey,

Thank you for reaching out. I'm working on the doc update and will let you know when I've added the limitation.

Sorry for the delay in response--I was out of the office the last two days.

Thanks again,
Ben

**Comment by netapp-bcammett** [comment]
Joey,

I published the limitation:
https://docs.netapp.com/us-en/occm/concept_worm.html#limitations

Thanks again for your feedback!

**Comment by netapp-joey** [comment]
Hey Ben,
Really appreciate your assistance in making this change!
Unfortunately, I made a typo in my original comment which I edited / corrected shortly after, but it appears the original was published.
Currently the 2nd sentence in the new bullet says "While WORM files and snapshots are protected....", but it should be just "While WORM files are protected..."
Apologies for the mix-up. It should be OK in the short-term but let me know if I can do anything to make fixing this easier.
Thanks again
